docs/2.guide/2.directory-structure/3.app.md

```diff
@@ -34,7 +34,7 @@ If you have a [`pages/`](/docs/guide/directory-structure/pages) directory, to di
 ```
 
 ::alert{type=danger}
-Since Nuxt 3 uses [`<Suspense>`](https://vuejs.org/guide/built-ins/suspense.html#suspense) inside `<NuxtPage>`, it cannot be set as a root element.
+Since Nuxt 3 uses [`<Suspense>`](https://vuejs.org/guide/built-ins/suspense.html#suspense) inside `<NuxtPage>`, `<NuxtPage>` cannot be set as a root element.
```

Comment:
Slight variation:
```suggestion
Since `<NuxtPage>` internally uses the [`<Suspense>`](https://vuejs.org/guide/built-ins/suspense.html#suspense) component, it cannot be set as a root element.
```

Reply:
Why reintroduce the ambiguous "it" that I intend to clarify :joy_cat: For devs not knowing how `Suspense` works, they could confuse it with `NuxtPage` - or at least be unsure about it.

Reply:
I do agree it was confusing in the original sentence, as the focus was put on 'Nuxt 3', which we don't even need to mention in this case. I find the rephrased one easily understandable, but that is certainly a subjective matter :)

Linking to the Vue docs addresses the risk of being unsure about what the `<Suspense>` component is. I believe that's sufficient.

Reply:
I think just calling things by their name is as clear as it can get and not expensive. I like your proposal very much - if we could replace "it" with what it is I'd love to commit that!

Reply:
Here you are, edited my suggestion 😄